modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+.. Wait for the status to transition to *Active*. 
++
+In most cases, the upgrade takes less than 10 minutes. When the status transitions to *Active*, and the version of the Runtime Fabric reflects the new version, the upgrade is complete.
++
+If the Runtime Fabric and Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
++
+Keep your browser session open to use in later steps.
++
 [IMPORTANT]
 To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, upgrade to 1.4.54 before upgrading to 1.5.
+. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version:
+.. Using your terminal on a controller node, run the following command: 
++
+----
+cd /opt/anypoint/runtimefabric
+sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
++
+.. Change file permissions for the `rtfctl` binary: 
++
+----
+sudo chmod +x rtfctl
+----
+. If you are not using the latest version of the cluster-level components (installer package), upgrade to the latest version:
+
+.. Make sure the `/opt/anypoint/runtimefabric/installer` directory has a minimum of 4 GiB of disk space available.
+.. Verify you have root privileges. Root privileges are required to perform an upgrade.
+.. Verify your environment meets the prerequisites:
+
+*** A production configuration of Runtime Fabric, with a minimum of 3 controller VMs, is required.
+*** The internal load balancer must be running on at least 3 replicas to maintain availability.
+*** An external load balancer must be configured to load balance incoming requests to the controller VMs, with a health check configured for TCP port 443.
+*** The CPU and memory resources for at least one worker node must be available in the cluster.
++
+This enables safe removal of a worker node from the cluster to apply upgrades without impacting application availability.
+*** Applications serving inbound requests must be scaled to a minimum of 2 replicas.
++
+[WARNING] Runtime Fabrics running on cluster version 1.0.x encounter application downtime if configured to use an HTTP proxy. You must apply the HTTP proxy configuration on each node after the upgrade. Locate the Runtime Fabric version by running `rtfctl version` on a node and referencing the appliance version.
+
+.. Update the Mule deployments running on Runtime Fabric to the latest patch version.
+... Using your web browser, navigate to the *Applications* tab in Runtime Manager.
+... Select an application and select *Manage Application*. 
+... The value in the *Runtime version* field indicates if an update is available. Select the updated version of Mule runtime engine.
+... Select *Deploy* to redeploy the application with the updated Mule runtime engine version.
+... Repeat this for each application running on this Runtime Fabric.
+.. Verify Runtime Fabric is healthy:
```

Comment:
There is a subtle usage difference between "verify" and "verify that" and "ensure" and "ensure that"; see the style reference.
```suggestion
.. Verify that Runtime Fabric is healthy:
```